samples/client/petstore/zapier/operations/actions.js

```diff
@@ -1,9 +1,9 @@
 const PetApi = require('../apis/PetApi');
 const StoreApi = require('../apis/StoreApi');
 const UserApi = require('../apis/UserApi');
-const { searchMiddleware, hasSearchRequisites, isSearchAction } = require('../utils/utils');
+const { triggerMiddleware, isTrigger, searchMiddleware, hasSearchRequisites, isSearchAction } = require('../utils/utils');
 
-const actions = {
+const operations = {
```

Comment:
@a-wallen is this a breaking change? shall we keep the const name as `actions`?

Reply:
We can keep the name. Also we need to drop the commit that says (EM Only)

Reply:
Thank you!

Reply:
https://github.com/OpenAPITools/openapi-generator/pull/19643/commits/435c73dbb9c69f554c3e5625c862e6ff5389656e this might be the commit that you want to land on it's own. You could start from this PR https://github.com/OpenAPITools/openapi-generator/pull/19643.

Sorry that I can't generate the samples on my computer. Git Bash generates the wrong path separator on Windows...

Reply:
Looks like I was also able to update the samples properly by using the devcontainer in https://github.com/OpenAPITools/openapi-generator/pull/19643/commits/86307de0470d830ad3ffc370294ca08cbadcc880!

Reply:
nice.

i merged via https://github.com/OpenAPITools/openapi-generator/pull/19645

in which you're listed as co-author

thanks again for the PR